Source/WTF/wtf/PlatformHave.h

```diff
@@ -1074,6 +1074,9 @@
 
 #if PLATFORM(COCOA)
 #define HAVE_WEBGPU_IMPLEMENTATION 1
+#if !defined(HAVE_WEBGPU_SWIFT)
```

Comment:
Let's use ENABLE here. ENABLE will sometimes be false even on platforms that HAVE the capability.